docs/docs-content/clusters/cluster-management/certificate-management.md

```diff
@@ -27,6 +26,17 @@ years.
 
 :::
 
+## Certificate Renewal Impact on Cluster Availability
+
+Both automatic and manual renewal leverage the `kubeadm certs renew` command. For more information about
```

Comment:
this is not accurate. for k3s and rke2, we restart the k8s rke2-server/k3s-server. 
this does not impact workloads, but may impact cluster management availability during restart, depending on the distro.
you should state that we use different methods depending on the distro, but that the user doesn't need to do anything